src/transformers/modeling_utils.py

```diff
@@ -5168,6 +5168,9 @@ def tplize(mod: torch.nn.Module) -> None:
 
     @property
     def loss_function(self):
+        if hasattr(self, "_loss_function"):
+            return self._loss_function
+
```

Comment:
@ArthurZucker this was needed to be added for a few models that don't need everything the loss func was up to. Case was `xglm`